### DISCUSSION TITLE
Astro Framework

### DISCUSSION THREAD

**Discussion by Mike1969a** [mdbootstrap/TW-Elements#2303] [Support from community]
Hello,
I am new in using tw-elements and with astro framework. It does not work really for me. Does anyone have a small example how to use, how to implement tw elements in astro framework.
Thank you in advance.

**Comment by iprzybysz** [comment]
Hi @Mike1969a. we don't have an integration tutorial for this framework, but I recommend you to check our other [Integration Tutorials,](https://tw-elements.com/docs/standard/integrations/angular-integration/) maybe one of them will be helpful - e.g. [Next.js integration](https://tw-elements.com/docs/standard/integrations/next-integration/).

**Comment by vela91** [comment]
same issue, no work in astrojs :disappointed:
